### PR TITLE
meta: Enforce consistent styles

### PR DESCRIPTION
**Pull request by xla**
When reading through the codebase it is apparent that there are glaring
inconsistencies when it comes to imports and other smaller style
questions. This makes it harder and more time consuming navigating and
altering the code.

This might be a highly controversial change, depending on your personal
believes.

Signed-off-by: xla <self@xla.is>

~~* [ ] Referenced an issue explaining the need for the change~~
~~* [ ] Updated all relevant documentation in docs~~
~~* [ ] Updated all code comments where relevant~~
~~* [ ] Wrote tests~~
~~* [ ] Added entry in `.changelog/`~~

### CODE REVIEW

File: tendermint/src/validator.rs [hunk 1]
```diff
@@ -340,29 +344,34 @@ mod tests {
...
+        // strings.Join(strings.Split(fmt.Sprintf("%v", pk2.PubKey().Bytes()), " "), ", "))
+        // fmt.Println("pk3: ", strings.Join(strings.Split(fmt.Sprintf("%v", pk3.PubKey().Bytes()),
+        // " "), ", ")) fmt.Println("not_in_set: ",
+        // strings.Join(strings.Split(fmt.Sprintf("%v", not_in_set.PubKey().Bytes()), " "), ", "))
+        // v1 := types.NewValidator(pk1.PubKey(), 148151478422287875)
+        // v2 := types.NewValidator(pk2.PubKey(), 158095448483785107)
+        // v3 := types.NewValidator(pk3.PubKey(), 770561664770006272)
+        // set := types.NewValidatorSet([]*types.Validator{v1, v2, v3})
+        // fmt.Println("Hash:", strings.Join(strings.Split(fmt.Sprintf("%v", set.Hash()), " "), ",
+        // ")) }
```
Comment (thanethomson) [inline tendermint/src/validator.rs:374]: This is a little weird.

File: tendermint/src/vote/sign_vote.rs [hunk 2]
```diff
@@ -153,34 +156,32 @@ mod tests {
...
+        // PartSetHeader: types.PartSetHeader{
+        // Total: 1000000,
+        // Hash:  []byte("0022446688AACCEE1133557799BBDDFF"),
+        // },
+        // },
+        // ValidatorAddress: []byte{0xa3, 0xb2, 0xcc, 0xdd, 0x71, 0x86, 0xf1, 0x68, 0x5f, 0x21,
+        // 0xf2, 0x48, 0x2a, 0xf4, 0xfb, 0x34, 0x46, 0xa8, 0x4b, 0x35}, ValidatorIndex: 56789}
+        // signBytes := types.VoteSignBytes("test_chain_id", vote.ToProto())
+        // fmt.Println(strings.Join(strings.Split(fmt.Sprintf("%v", signBytes), " "), ", "))
+        // }
```
Comment (thanethomson) [inline tendermint/src/vote/sign_vote.rs:184]: Also weird.

File: tendermint/src/vote/sign_vote.rs [hunk 3]
```diff
@@ -234,34 +235,32 @@ mod tests {
...
```
Comment (thanethomson) [inline tendermint/src/vote/sign_vote.rs:263]: Same. Is there any way to tell the formatter to ignore a particular block comment?

**Review by thanethomson — COMMENTED** [review]
Overall I don't think this is controversial at all and looks great to me. There are a few minor cases, however, where it does result in poorer legibility (embedding Go code in Rust comments), but perhaps there's a better way to achieve the goals in those cases?

**Comment by xla** [comment]
@thanethomson Addressed the go block you commented on, generally the `#[rustfmt::skip]` directive seem to work for almost all blocks rustfmt touches.

**Review by thanethomson — APPROVED** [review]

**Pull request merged**